webpack.config.js

```diff
@@ -252,7 +252,7 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 						{
 							loader: 'postcss-loader',
 							options: {
-								plugins: [ require( 'autoprefixer' ) ],
+								plugins: [ require( 'postcss-custom-properties' )(), require( 'autoprefixer' ) ],
```

Comment:
why the `()` after the require?

Reply:
I'd try this:
1. Load just `postcss-loader` with no options whatsoever. It should pick up a config file.
2. Check if the `.postcssrc.json` config file is loaded. If it's not, try renaming to `postcss.config.js` or some other supported name :)
3. Check that the legacy `node-sass` pipeline is not broken. I suspect that the `build-css:sections` task already is broken.

Reply:
> why the () after the require?

just tried something